src/Microsoft.TemplateEngine.Cli/TemplatePackageCoordinator.cs

```diff
@@ -425,20 +425,30 @@ private async Task<(NewCommandStatus, Dictionary<IManagedTemplatePackageProvider
             NewCommandStatus result = NewCommandStatus.Success;
             IReadOnlyList<IManagedTemplatePackage> templatePackages = await _templatePackageManager.GetManagedTemplatePackagesAsync(false, cancellationToken).ConfigureAwait(false);
 
-            List<string> parsedIdentifiers = new List<string>();
-            foreach (string entry in commandArgs.TemplatePackages)
+            List<Tuple<string, string>> parsedIdentifiers = new List<Tuple<string, string>>();
```

Comment:
It would be good to name properties in tuple (string PackageId, string ExpandedPackageId)